pyo3-introspection/src/introspection.rs

```diff
@@ -132,6 +133,12 @@ fn find_introspection_chunks_in_macho(
     if !macho.little_endian {
         bail!("Only little endian Mach-o binaries are supported");
     }
+    ensure!(
+        !macho.load_commands.iter().any(|command| {
+            matches!(command.command, CommandVariant::DyldChainedFixups(_))
+        }),
+        "Macho-o binaries with fixup chains are not supported yet, to avoid using fixup chains, use `--codegen=link-arg=-no_fixup_chains` option"
```

Comment:
Macho-O? :D

Reply:
Thanks!

```suggestion
        "Mach-O binaries with fixup chains are not supported yet, to avoid using fixup chains, use `--codegen=link-arg=-no_fixup_chains` option"
```